internal/riverinternaltest/riverinternaltest_test.go

```diff
@@ -16,7 +16,7 @@ import (
 // `riverinternaltest_test`, except that the test below uses internal variables
 // like `dbPool`.
 type Executor interface {
-	Exec(context.Context, string, ...interface{}) (pgconn.CommandTag, error)
+	Exec(ctx context.Context, query string, args ...interface{}) (pgconn.CommandTag, error)
```

Comment:
Like this one too. The nice part about having default param names is that when you tell your IDE to autocomplete an implementation scaffold, it can give you nice-looking names in it instead of something generically generated.